accounts/keystore/keystore_passphrase.go

```diff
@@ -97,7 +102,25 @@ func (ks keyStorePassphrase) StoreKey(filename string, key Key, auth string) err
 	if err != nil {
 		return err
 	}
-	return writeKeyFile(filename, keyjson)
+	// Write into temporary file
+	tmpName, err := writeTemporaryKeyFile(filename, keyjson)
+	if err != nil {
+		return err
+	}
+	if !ks.skipKeyFileVerification {
+		// Verify that we can decrypt the file with the given password.
+		_, err = ks.GetKey(key.GetAddress(), tmpName, auth)
+		if err != nil {
+			msg := "An error was encountered when saving and verifying the keystore file. \n" +
+				"This indicates that the keystore is corrupted. \n" +
+				"The corrupted file is stored at \n%v\n" +
+				"Please file a ticket at:\n\n" +
+				"https://github.com/klaytn/klaytn/issues" +
+				"The error was : %s"
+			return fmt.Errorf(msg, tmpName, err)
+		}
+	}
+	return os.Rename(tmpName, filename)
```

Comment:
How about changing above like below? Negation of boolean value could look confusing sometimes.
```suggestion
	if ks.skipKeyFileVerification {
	        return os.Rename(tmpName, filename)
	}
	// Verify that we can decrypt the file with the given password.
	_, err = ks.GetKey(key.GetAddress(), tmpName, auth)
	if err != nil {
		msg := "An error was encountered when saving and verifying the keystore file. \n" +
			"This indicates that the keystore is corrupted. \n" +
			"The corrupted file is stored at \n%v\n" +
			"Please file a ticket at:\n\n" +
			"https://github.com/klaytn/klaytn/issues" +
			"The error was : %s"
		return fmt.Errorf(msg, tmpName, err)
	}
	
```

Reply:
I removed negation representation, but in different way! (https://github.com/klaytn/klaytn/pull/725/commits/c7449d3210b94560bb140af060670563bb58f663)
`if !ks.skipKeyFileVerification` ->`if ks.skipKeyFileVerification == false`